ql/src/test/queries/clientnegative/jdbc_table_create_with_wrong_password.q

```diff
@@ -0,0 +1,12 @@
+--! qt:database:mariadb:q_test_country_table_with_schema.mariadb.sql
+
+-- Create jdbc table with wrong password(hive.sql.dbcp.password)
+CREATE EXTERNAL TABLE country_test1 (id int, name varchar(20))
```

Comment:
Would it be possible to write unit tests using derby which are more lightweight than those starting up docker containers?

Reply:
Good suggest. We already have the derby qtest which can verify this change, **external_jdbc_negative.q**. But this test has been disabled as its flaky. Do you think we need reopen this test? thanks.